packages/web/src/common/store/player/sagas.ts

```diff
@@ -132,7 +132,11 @@ export function* watchPlay() {
 
       const endChannel = eventChannel((emitter) => {
         audioPlayer.load(
-          track.track_segments,
+          track.duration ||
```

Comment:
Are there cases where duration won't be there now that we have added the duration field in the AudiusBackend side? Or are there cases where we haven't covered on api adapter side that this handles?

Reply:
duration should always be there in discovery (assuming the node upgraded / ran the new migration). it's possible I missed a spot in client. it would be 0 for all tracks up until now, though

Reply:
Okay cool prob fine to have the dupe